index.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width" />
+    <title>Mobile</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <div class="header">
+      <a class="name" href="assets/images/Vahan.jpg"
+        ><p>Vahan Khachvankian</p></a
+      >
+      <div class="burger">
+        <a href="#"><img src="assets/images/lines-56763.png" alt="logo" /></a>
+      </div>
+    </div>
+
+    <div class="main">
+      <div class="title">
+        <h1>
+          Hey There, <br />
```

Comment:
- [ ] Please correct this typo 😸 

```suggestion
          Hey There. <br />
```

Reply:
Done.